### ISSUE TITLE
AppImage Crashing

### ISSUE THREAD

**Issue by masx14** [ahrm/sioyek#11]
When I try to open specific files with the Sioyek Appimage it immediatly crashes:

```
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
"/tmp/.mount_SioyekLyVQ11/usr/bin/prefs.config"
"/tmp/.mount_SioyekLyVQ11/usr/bin/keys.config"
"/home/max/.local/share/Sioyek/prefs_user.config"
"/home/max/.local/share/Sioyek/keys_user.config"
"/home/max/.local/share/Sioyek/test.db"
"/home/max/.local/share/Sioyek/tutorial.pdf"
"/home/max/.local/share/Sioyek/last_document_path.txt"
"/tmp/.mount_SioyekLyVQ11/usr/bin/shaders"
config file change listener constructor called1starting index thread ...
Compiling shader : 
Compiling shader : �jYMV
Compiling shader : �YMV
Compiling shader : �YMV
Compiling shader : MV
Compiling shader : 03ZMV
Compiling shader : ��ZMV
Compiling shader : PEZMV
Compiling shader : ��ZMV
Compiling shader : @�ZMV
Compiling shader : @[MV
Compiling shader : �[MV
texture: 5
texture: 6
figure indexing finished ... 
getting links .... for 12
config file change listener constructor called1getting links .... for 13
config file change listener destructor called1invalid command; resetting to root
terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
  what():  filesystem error: Cannot convert character sequence: Ungültiges oder unvollständiges Multi-Byte- oder Wide-Zeichen

```

I have narrowed the problem down to mutated vowels. 

The name of the file was: "Schönleber Verfahren zur Charakterisierung des Niederfrequenzverhaltens von.pdf"

changed filename : "Schonleber Verfahren zur Charakterisierung des Niederfrequenzverhaltens von.pdf"

since I changed the filename, it didn't crash anymore

**Comment by ahrm** [comment]
Hello! thank you for your bug report.

I think this is a bug with g++-9 which I use to compile the linux version (the windows version works fine). See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=95048 .

I'll see if I can change the compiler.

**Comment by ahrm** [comment]
I think this is fixed in 6dce273912a42acfbc9c4c0a369c620a777a14f3. Here is an experimental build of that commit:

https://github.com/hexomancer/sioyek/releases/download/v0.31.63/sioyek-release-linux-portable.zip
